### PR TITLE
tailcfg: bump capver for 1.38

### PR DESCRIPTION
**Pull request by maisem**

### CODE REVIEW

**Review by DentonGentry — APPROVED** [review]

**Review by andrew-d — APPROVED** [review]

File: tailcfg/tailcfg.go [hunk 1]
```diff
@@ -94,7 +94,8 @@ type CapabilityVersion int
 //   - 55: 2023-01-23: start of c2n GET+POST /update handler
 //   - 56: 2023-01-24: Client understands CapabilityDebugTSDNSResolution
 //   - 57: 2023-01-25: Client understands CapabilityBindToInterfaceByRoute
-const CurrentCapabilityVersion CapabilityVersion = 57
+//   - 58: 2023-03-10: Client retries lite map updates before restarting map poll.
```
Comment (bradfitz) [inline tailcfg/tailcfg.go:97]: This isn't really a typical capver bump description. The client can't do anything new. Its behavior changed, sure, but will the server do anything differently as a result of this?

Comment (maisem) [reply tailcfg/tailcfg.go:97]: I added some tracking metrics, and wanted to correlate them with the change. I can change the description to only be `nothing, bump for 1.38`

Comment (andrew-d) [reply tailcfg/tailcfg.go:97]: If something goes wrong with these changes, I like having this as a distinct capver so we can have the server ignore the `ReadOnly` flag or force-restart map polls or whatever until we fix the issue.

Comment (bradfitz) [reply tailcfg/tailcfg.go:97]: sure, s'fine

Comment (bradfitz) [reply tailcfg/tailcfg.go:97]: But remember we never bump capver on branches. So if we have to fix the issue in the client, we can't distinguish 1.38.0 from 1.38.1 using capver alone.

**Pull request merged**